### ISSUE TITLE
Je potřeba umět zaznamenávat historii změn

### ISSUE THREAD

**Issue by jakubklimek** [apolicky/ds-cme#3]
Je potřeba umět zaznamenávat historii změn. Možná to udělá Dataspecer API.

**Comment by sstenchlak** [comment]
Ano, toto by mělo být pokryté na straně Dataspeceru. Samozřejmě s návrhem a implementací může přispět kdokoli.

Podobný princip se pak využije i pro strukturální model a bude z něj vycházet i evoluce.
